_posts/2023-11-07-routines.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Share your best Trino SQL routine"
+author: "Martin Traverso, Dain Sunstrom, David Phillips, Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/trino-sql-routine.png
+show_pagenav: false
+---
+
+We want to see the best [SQL routines]({{site_url}}/docs/current/routines.html)
+you can write, feature them as [examples in the
+documentation]({{site_url}}/docs/current/routines/examples.html), and send you
+some goodies as a reward!
+
+<!--more-->
+
+With the recent [Trino 431
+release]({{site_url}}/docs/current/release/release-431.html) we shipped a
+feature that has been awaited by many Trino users for a long, long time. [SQL
+routines]({{site_url}}/docs/current/routines.html) are an easy way to define our
+own procedural, custom function. All users of your Trino instance can then use
+that function in their queries and enjoy the new feature to simplify their
+queries.
+
+The new process of simply writing a routine in your client tool in SQL, replaces
+the old way of having to create a custom plugin in Java, compiling it, and
+getting the binary deployed in your cluster. Time to using a function went from
+hours to minutes!
+
+Our documentation includes details for all the supported statements:
+
+* `BEGIN`
+* `CASE`
+* `DECLARE`
+* `FUNCTION`
+* `IF`
+* `ITERATE`
+* `LEAVE`
+* `LOOP`
+* `REPEAT`
+* `RETURN`
+* `SET`
+* `WHILE`
+
+With the memory connector and the Hive connector supporting routine storage, you
+can use `CREATE FUNCTION` and `DROP FUNCTION`, so that everyone using the
+cluster has access to your routines.
+
+The unit tests and our examples documentation contain a number of routines that
```

Comment:
```suggestion
The unit tests and our example documentation contain a number of routines that
```
Link to the documentation, and maybe even unit tests on GitHub?

Reply:
Not the tests .. but sure.. the docs